modules/virt-deleting-vmis-cli.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-deleting-vmis-cli.adoc
+
+[id="virt-deleting-vmis-cli_{context}"]
+
+= Deleting a standalone virtual machine instance using the CLI
+
+You can delete a standalone virtual machine instance (VMI) by using the `oc` command-line interface (CLI).
+
+.Prerequisites
+
+* Identify the name of the VMI that you want to delete.
+
+.Procedure
+
+* Delete the VMI by running the following command:
++
+----
+$ oc delete vmi <vmi_name>
+----
++
+[NOTE]
```

Comment:
I think this `NOTE` is not necessary, it should be a common knowledge.

Reply:
This note was there in the content written by the previous writer. I believe it would be helpful to novice users, and is not harmful if this note is present. If you want this note removed, I will check with  the author who created this note in the previously shipped releases. Please advise.

Reply:
It's can be helpful, but it is a very common note. Actually, every command with `oc` can have this kind of note, but not necessary.

Reply:
As per your guidance, I am removing this note. You indicate that this particular note is common knowledge among users, and it applies to almost all "oc" commands, therefore my understanding is that it is redundant and not helpful to have this note in this particular location. I am adding this comment so that if a question arises during peer review as to why this note (which existed in the previous release) is being removed, they would know the context and reason for removing the note. Thanks.